backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/NamespaceImpl.java

```diff
@@ -75,7 +80,15 @@ protected EClass eStaticClass() {
      */
     @Override
     public EList<Membership> getImportedMembership() {
-        List<Membership> importedMemberships = new ArrayList<>();
+        return getImportedMembership(new BasicEList<Namespace>());
+    }
+
+    private EList<Membership> getImportedMembership(EList<Namespace> excluded) {
```

Comment:
Please add a @generated NOT for this method.

Please check all other modified/new methods in this PR and add @generated NOT if needed.

Reply:
Done